digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java

```diff
@@ -2,36 +2,71 @@
 
 import com.google.common.base.Optional;
 import io.digdag.client.config.Config;
-import io.digdag.client.config.ConfigException;
 import io.digdag.core.storage.StorageManager;
 
-import java.util.Arrays;
 import java.util.List;
 
 public class EcsClientConfig
 {
     private static final String SYSTEM_CONFIG_PREFIX = "agent.command_executor.ecs.";
+    public static final String TASK_CONFIG_ECS_KEY = "agent.command_executor.ecs";
```

Comment:
This const value is assumed to be used to set the nested config for ecs client config at `taskConfig` as follows.

```java
taskConfig.set(EcsClientConfig.TASK_CONFIG_ECS_KEY, ecsClientConfig)
```